docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/00_Prerequisites_And_Assumptions/Database_Configuration.md

```diff
@@ -28,3 +28,41 @@ GRANT ALTER, DROP, CREATE, UPDATE, INSERT, SELECT, DELETE, CREATE TEMPORARY TABL
 within the `adminUser` and `adminPassword` tags.
+
+Tips:
+
+* ConfigSetting host value should not terminate in a slash. E.g. "http://localhost" not "http://localhost/". To fix, run: (for http) UPDATE Config SET Value='http://localhost' WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='host');
+* If you get an error after clicking "Submit" or "Save data" on a form, check that url Config setting is set for your host. (Previous iterations of LORIS recommended setting this to the empty string) Run: UPDATE Config SET Value='_$yourhost_' WHERE ConfigID=(SELECT ID FROM ConfigSettings WHERE Name='url');
+* If your dashboard loads but no other modules load, ensure that your apache config file (in CentOS: /var/apache2/apache2.conf) is set to AllowOverride All in the section <Directory /var/www/> to enable re-write rules (based on htdocs/.htaccess)
+* ensure your smarty/templates_c directory is writable by apache
+* run composer update to ensure your dependencies (including smarty) are up-to-date
+
+## Create Front-end users
+
+The installation script adds admin user/password, associated with a dummy email address which should be changed. Additional users and permissions are added in the User Accounts page. For clarity and uniqueness, it is recommended to use an email address as username.  
+
+* To set/reset a user password, use the script [tools/resetpassword.php](https://github.com/aces/Loris/blob/main/tools/resetpassword.php)  
+
+```SQL
+GRANT ALTER, DROP, CREATE, UPDATE, INSERT, SELECT, DELETE, CREATE TEMPORARY TABLES, LOCK TABLES  on $dbname.* to 'lorisDBadmin'@'$dbhost' IDENTIFIED BY '$newpassword' WITH GRANT OPTION ;
+```
+
+lorisadmin is the Loris administrator Unix account. Developers may wish to have their own individual accounts.
+
+## Create back-end accounts
+
+Two types of back-end accounts are useful for administering Loris: MySQL accounts and Unix account.
+
+It is good practice to create a new MySQL user for developer purposes (e.g. lorisDBadmin) to execute all back-end transactions, both in configuration and in day-to-day operations (i.e. not MySQL root). This account should be distinct from the MySQL root user as well as distinct from the lorisuser account which the install script created with limited permissions for executing all transactions coming from Loris' PHP code (i.e. from front-end users' activity).
+
+## Backups
+
+Database backups are imperative. There are many ways to automate the creation of backups, using bash tools such as rsync and taking advantage of cronjobs.
+
+One example set-up is to perform a `mysqldump` once every week. To do this, open `/etc/crontab` with a text editor and add the following line:
+
+```
+0 0 0 0 sun mysqldump --user=USER --host=DBHOSTURL --password=PASSWORD DBNAME > /home/lorisadmin/backups/`date +%Y-%m-%d.sql`;
+```
+
+This will create a weekly backup in `~/backups`. This should be copied over to `/data/$project/data` to preserve hard disk space.
```

Comment:
This is not a requirement at all and is just adding steps to an already complex process. I don't think it's LORIS' job to educate system administrators on the importance of backups otherwise you might as well add a section on how to clean your computer and make sure to wire you cables properly... If anyone insists on doing it, please put it in the developers or system administrator best practices.